app/workers/edition_batch_link_checker.rb

```diff
@@ -0,0 +1,31 @@
+# Calls the Link Checker API to verify all links in an edition per organisation
+require "sidekiq-scheduler"
+
+class EditionBatchLinkChecker
+  include Sidekiq::Worker
+  include Rails.application.routes.url_helpers
+
+  def perform(organisation_id = nil)
+    organisation = find_organisation(organisation_id)
```

Comment:
I wonder why we need to make it possible to pass in the organisation here, the only place it's used right now it will always be `nil`. We could add support for specifying organisations later if we need it.